### PR TITLE
Add Windows tab for some commands

### PR DESCRIPTION
**Pull request by meowmeowmeowcat**
Some commands are missing Windows tabs so I've added them.

### CODE REVIEW

File: source/guides/installing-stand-alone-command-line-tools.rst [hunk 1]
```diff
@@ -22,10 +22,19 @@ packages, and allows you to safely run the program from anywhere.
 
 ``pipx`` is installed with ``pip``:
 
-::
+.. tab:: Unix/macOS
+
+  .. code-block:: bash
+
+      $ python3 -m pip install --user pipx
+      $ python3 -m pipx ensurepath  # ensures the path of the CLI application directory is on your $PATH
```
Comment (bhrutledge) [inline source/guides/installing-stand-alone-command-line-tools.rst:30]: Can we expand the scope of this PR to normalize the way the input is displayed in this document? For example:

```suggestion
      python3 -m pip install --user pipx

      python3 -m pipx ensurepath  # ensures the path of the CLI application directory is on your $PATH
```

Comment (meowmeowmeowcat) [reply source/guides/installing-stand-alone-command-line-tools.rst]: Sure.

**Comment by meowmeowmeowcat** [comment]
Or maybe we should follow the [suggestion](https://github.com/pypa/packaging.python.org/pull/937#discussion_r667786903) of @webknjaz (adding prompts for command) so folks won't get confused?

File: source/contribute.rst [hunk 2]
```diff
@@ -79,7 +79,7 @@ need:
 1. `Nox <https://nox.readthedocs.io/en/latest/>`_. You can install or upgrade
    nox using ``pip``::
 
-      python -m pip install --user nox
+      python3 -m pip install --user nox
```
Comment (webknjaz) [inline source/contribute.rst:82]: This doesn't look related to having windows tabs.

File: source/guides/installing-stand-alone-command-line-tools.rst [hunk 3]
```diff
@@ -45,6 +57,7 @@ For example
     These binaries are now globally available
       - cowsay
   done! ✨ 🌟 ✨
+
```
Comment (webknjaz) [inline source/guides/installing-stand-alone-command-line-tools.rst:60]: Don't do this. If it's one terminal session, it's unnatural to have extra whitespace before each prompt.

Comment (webknjaz) [reply source/guides/installing-stand-alone-command-line-tools.rst:60]: Instead, use the `console` lexer to highlight it properly.

File: source/contribute.rst (hunk 2 above)
Comment (meowmeowmeowcat) [reply source/contribute.rst:82]: I want to update `python` to `python3` at the same time. However, is this should be fixed in another PR?

File: source/guides/installing-stand-alone-command-line-tools.rst (hunk 3 above)
Comment (meowmeowmeowcat) [reply source/guides/installing-stand-alone-command-line-tools.rst:60]: Okay

File: source/contribute.rst (hunk 2 above)
Comment (webknjaz) [reply source/contribute.rst:82]: That sounds like a separate change to me. The rule of thumb is to make PRs (and individual commits for that matter) atomic.
https://seesparkbox.com/foundry/atomic_commits_with_git

Comment (meowmeowmeowcat) [reply source/contribute.rst:82]: Thanks for these articles! :)
I will read them.

File: source/guides/installing-stand-alone-command-line-tools.rst [hunk 4]
```diff
@@ -22,39 +22,52 @@ packages, and allows you to safely run the program from anywhere.
...
+Now you can install packages with ``pipx install``::
+
+  pipx install PACKAGE
 
+And then access the package's entry point(s) from anywhere.
 ::
 
-  $ pipx install PACKAGE
-  $ ENTRYPOINT_OF_PACKAGE [ARGS]
+  ENTRYPOINT_OF_PACKAGE [ARGS]
```
Comment (webknjaz) [inline source/guides/installing-stand-alone-command-line-tools.rst:50]: This looks weird. I would never have guessed that it's a placeholder for a command w/o a prompt. Dunno what to do about it...

File: source/guides/using-testpypi.rst [hunk 5]
```diff
@@ -28,7 +28,7 @@ the ``--repository`` flag
 
 .. code::
 
-    $ twine upload --repository testpypi dist/*
+    twine upload --repository testpypi dist/*
```
Comment (webknjaz) [inline source/guides/using-testpypi.rst:31]: This prompt-removal crusade doesn't seem to be related to adding the windows tabs either

File: source/guides/installing-stand-alone-command-line-tools.rst [hunk 6]
```diff
@@ -22,39 +22,52 @@ packages, and allows you to safely run the program from anywhere.
...
-    ___
-  < moo >
-    ===
-          \
-           \
-             ^__^
-             (oo)\_______
-             (__)\       )\/       ||----w |
-                 ||     ||
+.. code-block:: console
```
Comment (webknjaz) [inline source/guides/installing-stand-alone-command-line-tools.rst:54]: This should probably go into a separate PR that adjusts the code session blocks.

File: source/guides/installing-stand-alone-command-line-tools.rst (hunk 4 above)
Comment (meowmeowmeowcat) [reply source/guides/installing-stand-alone-command-line-tools.rst:50]: Probably prompt should be remained here.

File: source/guides/using-testpypi.rst (hunk 5 above)
Comment (meowmeowmeowcat) [reply source/guides/using-testpypi.rst:31]: yes, but @bhrutledge started this 🤣 [here](https://github.com/pypa/packaging.python.org/pull/942#discussion_r667913835).

File: source/guides/installing-stand-alone-command-line-tools.rst (hunk 6 above)
Comment (meowmeowmeowcat) [reply source/guides/installing-stand-alone-command-line-tools.rst:54]: okay

File: source/guides/using-testpypi.rst (hunk 5 above)
Comment (webknjaz) [reply source/guides/using-testpypi.rst:31]: I see. Still, let's try to keep PRs dedicated to a single cause each. Where it's possible, of course :)

Comment (meowmeowmeowcat) [reply source/guides/using-testpypi.rst:31]: Okay, and I will revoke these changes and open a new PR for these. :)

File: source/guides/installing-stand-alone-command-line-tools.rst [hunk 7]
```diff
@@ -22,18 +22,31 @@ packages, and allows you to safely run the program from anywhere.
...
+      py -m pip install --user pipx
+      
+      py -m pipx ensurepath
 
 .. Note:: You may need to restart your terminal for the path updates to take effect.
 
-Now you can install packages with ``pipx install`` and access the package's entry point(s) from anywhere.
+Now you can install packages with ``pipx install``::
 
+  $ pipx install PACKAGE
```
Comment (webknjaz) [inline source/guides/installing-stand-alone-command-line-tools.rst:45]: If we're keeping the prompt, the `console` lexer should be used.

Comment (meowmeowmeowcat) [reply source/guides/installing-stand-alone-command-line-tools.rst:45]: yes, and this will be solved in another PR.

Comment (bhrutledge) [reply source/guides/installing-stand-alone-command-line-tools.rst]: Since this was the only remaining unrelated change, I reverted it 307732b.

File: source/guides/using-testpypi.rst (hunk 5 above)
Comment (bhrutledge) [reply source/guides/using-testpypi.rst]: Apologies for the confusion. I thought it would be a small amount of extra scope, that would improve consistency. I see how it was too much. Thanks @meowmeowmeowcat and @webknjaz for bearing with me.

**Review by bhrutledge — APPROVED** [review]
